compile.sh

```diff
@@ -0,0 +1 @@
+gcc src/fabtris2.c src/fabtrimino.c src/next.c -I./src -lSDL2 -lSDL2_image -g -o fabtris2
```

Comment:
Also never a bad idea to explicitly state which standard you're compiling against:
```
-std=gnu11
```